clientlibs/java/src/main/java/org/upgradeplatform/responsebeans/InitializeUser.java

```diff
@@ -4,24 +4,15 @@
 import java.util.Map;
 
 public class InitializeUser {
-    private String id;
```

Comment:
The response object still has the `id` in it so this is failing when I run the request. It seems like maybe they used to be identical but now we may not be able to use the response type to define the request.